internal/command/action_test.go

```diff
@@ -105,3 +107,30 @@ func TestReturnStatusOfError(t *testing.T) {
 		So(err, ShouldEqual, expected)
 	})
 }
+
+func TestPrintProviderVersions(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping long-running test.")
+	}
+
+	Convey("printProviderVersions should return a string of all of the providers in the DefaultPath", t, func() {
+		pathTo,_ := exec.Command("pwd").Output()
+		pathString := fmt.Sprintf("%s",pathTo)
+		pathString = strings.TrimSuffix(pathString, "\n")
+		pathString = strings.TrimSuffix(pathString, "\r")
+
+		exec.Command(path.Join(string(pathString), "testversions/setup.sh")).Run()
```

Comment:
This should be just writing a bash file to a tmp location - we don't need (nor would we want) any real providers. Bash files for testing can be as simple as:
```
#!/usr/bin/env bash
echo "v1.2.3"
```

Reply:
Changed and pushed

Reply:
I figured a real test environment was necessary